test/release/examples/benchmarks/shootout/Makefile

```diff
@@ -90,22 +90,22 @@ pidigits-fast: pidigits-fast.chpl
 	fi
 
 regexdna: regexdna.chpl
-	@+if [ "$(CHPL_REGEXP)" != "none" ]; then \
+	@+if [ "$(CHPL_RE2)" != "none" ]; then \
 	$(CHPL) -o $@ $(CHPL_FLAGS) $<; \
 	else \
 	echo "";\
-	echo "Can't build regexdna without CHPL_REGEXP being enabled. See for details:"; \
-	echo "https://chapel-lang.org/docs/usingchapel/chplenv.html#chpl-regexp"; \
+	echo "Can't build regexdna without CHPL_RE2 being enabled. See for details:"; \
+	echo "https://chapel-lang.org/docs/usingchapel/chplenv.html#chpl-regex"; \
```

Comment:
Seems like the tag at the end of the html address should reflect the environment variable.  Has that been updated or does this address still work?

Reply:
> Seems like the tag at the end of the html address should reflect the environment variable. Has that been updated or does this address still work?

`regexp` was changed to `regex` (case-preserved) almost everywhere, except that `CHPL_REGEXP` was changed to `CHPL_RE2` .

In the above documentation, the tag was changed to `chpl-re2` based on the section name's change so I will update the link accordingly.